src/UIKit/NSLayoutManager.cs

```diff
@@ -98,6 +107,24 @@ partial class NSLayoutManager {
 				}
 			}
 		}
+#endif // !XAMCORE_4_0
+
+		[Watch (6,0), TV (13,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
```

Comment:
Do you need to create this because it is not supported in C#? When do you know if you need to create an additional file?

Reply:
I need to create this because the **generator** does not know how to handle certain types and these need to be manually bound, meaning C# does support this it is just the generator does not know how to output the right C# code for these.

Generally the generator will yell at you when it does not know how to handle a type but basically C arrays and types that do not inherit from NSObject.